runtime/compiler/optimizer/VectorAPIExpansion.cpp

```diff
@@ -818,18 +898,15 @@ TR_VectorAPIExpansion::expandVectorAPI()
          TR::RecognizedMethod index = methodSymbol->getRecognizedMethod();
          int32_t handlerIndex = index - _firstMethod;
 
-         TR_ASSERT_FATAL(methodTable[handlerIndex]._methodHandler(this, treeTop, node, elementType, vectorLength, checkMode),
+         TR_ASSERT_FATAL(methodTable[handlerIndex]._methodHandler(this, treeTop, node, elementType, vectorLength, numLanes, checkMode),
                          "Analysis should've proved that method is supported");
 
          TR::Node::recreate(parent, TR::treetop);
-         methodTable[handlerIndex]._methodHandler(this, treeTop, node, elementType, vectorLength, doMode);
+         methodTable[handlerIndex]._methodHandler(this, treeTop, node, elementType, vectorLength, numLanes, doMode);
```

Comment:
The doScalarization code that follows the handler call is common for all handlers, so that's why the check. I did not want to duplicate it in each handler.